site-container.yml.sample

```diff
@@ -174,6 +174,60 @@
             status: "Complete"
             end: "{{ lookup('pipe', 'date +%Y%m%d%H%M%SZ') }}"
 
+- hosts: mons[0]
+  become: True
+  gather_facts: false
+  any_errors_fatal: true
+  tasks:
+    - import_role:
+        name: ceph-defaults
+
+    - name: set global config
+      ceph_config:
+        action: set
+        who: "global"
+        option: "{{ item.key }}"
+        value: "{{ item.value }}"
+      environment:
+        CEPH_CONTAINER_IMAGE: "{{ ceph_docker_registry + '/' + ceph_docker_image + ':' + ceph_docker_image_tag if containerized_deployment | bool else None }}"
+        CEPH_CONTAINER_BINARY: "{{ container_binary }}"
+      with_dict:
+        "{{ { 
+           'public_network': public_network | default(False),
+           'cluster_network': cluster_network | default(False),
+           'osd pool default crush rule': osd_pool_default_crush_rule,
+           'ms bind ipv6': 'true' if ip_version == 'ipv6' else 'false',
+           'ms bind ipv4': 'false' if ip_version == 'ipv6' else 'true',
+           'osd crush chooseleaf type': '0' if common_single_host_mode | default(False) | bool else False,
+        } }}"
+      when:
+        - inventory_hostname == ansible_play_hosts_all | last
+        - item.value
+
+    - name: set global config overrides
+      ceph_config:
+        action: set
+        who: "global"
+        option: "{{ item.key }}"
+        value: "{{ item.value }}"
+      environment:
+        CEPH_CONTAINER_IMAGE: "{{ ceph_docker_registry + '/' + ceph_docker_image + ':' + ceph_docker_image_tag if containerized_deployment | bool else None }}"
+        CEPH_CONTAINER_BINARY: "{{ container_binary }}"
+      when: inventory_hostname == ansible_play_hosts_all | last
+      with_dict: "{{ ceph_conf_overrides['global'] }}"
```

Comment:
We need to take `ceph_conf_overrides` into consideration for all daemons as well as some modules need to have configs before startup.